requirements.txt

```diff
@@ -5,4 +5,4 @@
 #    pip-compile --output-file requirements.txt requirements.in
 #
 
-requests==2.8.1
```

Comment:
If the project is intended to be used as a library, it's probably better to specify a relative version (`requests>=2.8.1`), so that we don't have to change this again when we upgrade `requests` in lc-recommendation.

We should submit the patch upstream as well.